tests.yml

```diff
@@ -0,0 +1,66 @@
+# Run python-versions on {{ taskotron_item }} NVR build
+
+- hosts: localhost
+  remote_user: root
+  vars:
+    testcase: dist.python-versions
+    taskotron_generic_task: true
+    # below are fallback vars for local testing
+    artifacts: ./artifacts
+    taskotron_item: python-gear-0.11.0-1.fc27  # you should really override at least this :)
+    taskotron_supported_binary_arches:
+      - x86_64
```

Comment:
Did we just get rid of all the other architectures?

Reply:
Again, that's a default variable value, for local execution when developing/experimenting. It is expected that you override this (and taskotron stack will do that for you, with the list of arches we want to execute on).

Reply:
ok